### PR TITLE
Applying the is_beta and verified flags on both the user's lists and graph data

### PR DESCRIPTION
**Pull request by LanternNassi**
# Description

Applying the is_beta and verified flags on both the user's lists and graph data

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Trello Ticket ID

https://trello.com/c/TknAIvEM/1647-admin-users-list-endpoints

## How Can This Be Tested?
It can be tested through fetching the users graph data and also the list of users

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/controllers/users.py [hunk 1]
```diff
@@ -123,15 +123,34 @@ def get(self):
...
+        All = request.args.get('All' , False)
 
         users = []
 
         if (keywords == ''):
-            paginated = User.find_all(paginate=True, page=page, per_page=per_page)
-            users = paginated.items
-            pagination = paginated.pagination
+            if not All :
+                paginated = User.query.filter(User.verified == verified , User.is_beta_user == is_beta).order_by(User.date_created.desc()).paginate(page=page, per_page=per_page, error_out=False)
```
Comment (MutegekiHenry) [inline app/controllers/users.py:134]: This part of the query will filter by those who are both verified and beta_user variables, what if one just wants to vary just one of the variables independently? E.g if I just want to know unverified users regardless of whether or not they are beta. So I would think there would be need to separate the filter queries for each scenario, what do you think?

Comment (LanternNassi) [reply app/controllers/users.py:134]: Oooh ok i had not looked at that angle when i was implementing the solution . Let me cater for it also . Thanks for that @MutegekiHenry

File: app/controllers/users.py [hunk 2]
```diff
@@ -150,7 +192,7 @@ def get(self):
         
         return dict(
             status='success',
-            data=dict(pagination=pagination, users=json.loads(users_data))
+            data=dict(meta_data = meta_data , pagination=pagination, users=json.loads(users_data))
```
Comment (Mubangizi) [inline app/controllers/users.py:195]: Change the meta data output to only show beta_users, non_verified and total_users something like;
`{
  beta_users: 131,
  non verified: 123,
  total_users: 3223
  }
`
.
Screenshot 2023-08-18 at 3 27 29 AM

**Review by Mubangizi — CHANGES_REQUESTED** [review]
Great Job Nessim, just a few minor tweaks to add here

**Review by MutegekiHenry — APPROVED** [review]
LGTM

**Review by Mubangizi — APPROVED** [review]
LGTM

**Pull request merged**